### PR TITLE
Palladio 495

### PR DESCRIPTION
**Pull request by rapp-fzi**
Please review and merge by considering the review notes in the ticket https://jira.palladio-simulator.com/browse/PALLADIO-495

### CODE REVIEW

File: bundles/de.uka.ipd.sdq.simucomframework/src/de/uka/ipd/sdq/simucomframework/SimuComFactory.java [hunk 1]
```diff
@@ -32,7 +34,8 @@
      *            Should desmoj trace our experiment
      * @return The created simulation model
      */
-    public static SimuComModel getSimuComModel(SimuComConfig config, SimuComStatus simuComStatus, boolean isRemote) {
+    public static SimuComModel getSimuComModel(SimuComConfig config, SimuComStatus simuComStatus, boolean isRemote
+            , IResourceTableManager resourceTableManager) {
```
Comment (sdkrach) [inline bundles/de.uka.ipd.sdq.simucomframework/src/de/uka/ipd/sdq/simucomframework/SimuComFactory.java:38]: The line break looks missplaced. I'd expect the comma to be on the previous line.

**Review by sdkrach — CHANGES_REQUESTED** [review]
The heavy argument passing is kind of ugly, but I think this is a more general problem. Small style fixe, otherwise LGTM.

File: bundles/de.uka.ipd.sdq.simucomframework/src/de/uka/ipd/sdq/simucomframework/SimuComFactory.java (hunk 1 above)
Comment (rapp-fzi) [reply bundles/de.uka.ipd.sdq.simucomframework/src/de/uka/ipd/sdq/simucomframework/SimuComFactory.java]: worked in review finding; reformatted file using Palladio code formatter

**Review by sdkrach — APPROVED** [review]
LGTM

**Pull request merged**